sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_event.cpp

```diff
@@ -16,13 +16,18 @@
 #include "ur_level_zero_event.hpp"
 
 void printZeEventList(const _ur_ze_event_list_t &UrZeEventList) {
-  urPrint("  NumEventsInWaitList %d:", UrZeEventList.Length);
+  if (UrL0Debug & UR_L0_DEBUG_BASIC) {
+    urPrint("  NumEventsInWaitList %d:", UrZeEventList.Length);
 
-  for (uint32_t I = 0; I < UrZeEventList.Length; I++) {
-    urPrint(" %#llx", ur_cast<std::uintptr_t>(UrZeEventList.ZeEventList[I]));
-  }
+    if (UrL0Debug & UR_L0_DEBUG_BASIC) {
```

Comment:
It looks like this "if" is redundant, could you please remove it if so.

Reply:
thanks @againull :) yes, missed that. removed